frontend/ui/src/pages/Inbox/Messenger/MessageList/index.tsx

```diff
@@ -165,12 +168,18 @@ const MessageList = (props: MessageListProps) => {
                   {formatDateOfMessage(message)}
                 </div>
               )}
-              <SourceMessage
-                source={getSource(conversation)}
-                message={message}
-                contact={contactToShow}
-                lastInGroup={lastInGroup}
-              />
+              <AvatarTime
```

Comment:
Somehow this name does not feel right for me? Maybe someone here has a better suggestion?

Reply:
always up for other suggestions :)

Reply:
`MessageWrapper` sounds super boring and generic...maybe `MessageRow` because this is what it actually should represent?